scripts/ci-check.sh

```diff
@@ -27,14 +27,17 @@ run() {
   fi
 }
 
-header "Code format and linting…"
+header "Lint files…"
 run make lint
 
-header "Run tests…"
-run make test-coverage
+header "Check code format…"
+run make check-format
 
-header "Run typecheck…"
-run make typecheck
+header "Run tests and check test code coverage…"
+run make check-code-coverage
+
+header "Check types…"
```

Comment:
I think we should have kept `Run typechecks` or even kept is `Typechecking…` as typecheck is the [idiomaticic verb](https://en.wiktionary.org/wiki/typecheck) for this operation in computing.

Reply:
You’re right! Since all other verbs are in the present tense, I’ll replace it with `"Typecheck files…"`.